applications/client/src/views/Campaign/Explore/Panels/Comment/Comments.tsx

```diff
@@ -216,38 +162,22 @@ export const Comments = observer<CommentsProps>(({ sort }) => {
 				}, 250);
 			}
 		}
-	}, [data, commentsData]);
+	}, [commandGroupIdData]);
 
 	return (
 		<VirtualizedList
 			rangeChanged={(visibleRange) => state.update('visibleRange', visibleRange)}
 			listRef={listRef}
 			cy-test="comments-view"
 		>
-			{store.router.params.currentItem === 'comments_list' ? (
-				store.campaign.commentsList.commandGroupIds.length === 0 ? (
-					<MessageRow>No comments</MessageRow>
-				) : isCommentsDataLoading ? (
+			{commandGroupIdData?.commandGroupIds?.length === 0 ? (
+				isLoading ? (
 					<ProgressBar intent={Intent.PRIMARY} />
 				) : (
-					store.campaign.commentsList.commandGroupIds.map((commandGroupId) => (
-						<CommentGroup
-							cy-test="comment-group"
-							key={commandGroupId}
-							commandGroupId={commandGroupId}
-							toggleNewComment={state.toggleNewComment}
-							newComment={state.newComment}
-							expandedCommandIDs={state.expandedCommandIDs}
-							removeExpandedCommandID={state.removeExpandedCommandID}
-						/>
-					))
+					<MessageRow>No comments</MessageRow>
 				)
-			) : data?.commandGroupIds.length === 0 ? (
-				<MessageRow>No comments</MessageRow>
-			) : isLoading ? (
-				<ProgressBar intent={Intent.PRIMARY} />
 			) : (
```

Comment:
> Using the gt.redeye file, I clicked into Explorer Mode, then the Comments tab. If you click on Favorited Comments, which has 0, the page looks like it's continuously loading but never does. We saw this in a previous PR and it was resolved, so whatever fixed that behavior must have been affected. No console errors.

[ ] caused by the new render logic in the `Comments` return. It doesn't account for if there are 0 comments.